api/src/main/java/org/pmiops/workbench/api/OfflineRuntimeController.java

```diff
@@ -30,23 +51,46 @@
 public class OfflineRuntimeController implements OfflineRuntimeApiDelegate {
   private static final Logger log = Logger.getLogger(OfflineRuntimeController.class.getName());
 
+  // On the n'th day of inactivity on a PD, a notification is sent.
+  private static final Set<Integer> INACTIVE_DISK_NOTIFY_THRESHOLDS_DAYS = ImmutableSet.of(14);
```

Comment:
This is a Set because I expect later we may want finer grained control over the initial email dates. By contrast, the period is a constant because multiple different periodic notification cycles are not going to make sense.